src/apps/MASS3DEA-Seq.cpp

```diff
@@ -0,0 +1,161 @@
+//~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~//
+// Copyright (c) 2017-23, Lawrence Livermore National Security, LLC
+// and RAJA Performance Suite project contributors.
+// See the RAJAPerf/LICENSE file for details.
+//
+// SPDX-License-Identifier: (BSD-3-Clause)
+//~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~//
+
+// Uncomment to add compiler directives for loop unrolling
+//#define USE_RAJAPERF_UNROLL
+
+#include "MASS3DEA.hpp"
+
+#include "RAJA/RAJA.hpp"
+
+#include <iostream>
+
+namespace rajaperf {
+namespace apps {
+
+void MASS3DEA::runSeqVariant(VariantID vid,
+                             size_t RAJAPERF_UNUSED_ARG(tune_idx)) {
+  const Index_type run_reps = getRunReps();
+
+  MASS3DEA_DATA_SETUP;
+
+  switch (vid) {
+
+  case Base_Seq: {
+
+    startTimer();
+    for (RepIndex_type irep = 0; irep < run_reps; ++irep) {
+
+      for (int e = 0; e < NE; ++e) {
+
+        MASS3DEA_0_CPU
+
+        CPU_FOREACH(iz, z, 1) {
```

Comment:
Do we need the outer z loop? It looks iz is not used in the loop body and the length of the outer loop is 1.